### PR TITLE
corrections to text hiring process

### PR DESCRIPTION
**Pull request by MirjamvT**
various changes for clarification and correction

### CODE REVIEW

File: activities/recruitment/hiring-process.md [hunk 1]
```diff
@@ -17,17 +17,17 @@ We publish all job postings on the [Publiccode.net](https://publiccode.net) webs
...
-6. Second interview: CEO with HRM, also partially based on the pre-established list, and with attention to issues not discussed in the first interview. Approximately. 80-90% of the time is spent on questioning the candidate. Immediately after the interview HRM and COO score the candidate, based on the list and other impressions gathered. HRM takes this input to CEO, and HRM and CEO determine if the candidate should receive an offer.
+5. First interview: a lead codebase steward and HRM, approximately 1 hr, partially on the basis of a pre-established list of desirable competences and other characteristics (i.e. fit for role, fit for organization (culture, growth, leadership, long/short term)). Half of the interview is spent presenting PublicCode.net, in the other half, the candidate has a chance to present her/himself. Immediately after the interview both HRM and CEO score the candidate on the items on the list and determine if the candidate should progress to a second interview. HRM will contact the candidate to inform her/him.
```
Comment (ElenaFdR) [inline activities/recruitment/hiring-process.md]:
```suggestion
5. First interview: a lead codebase steward and HRM, approximately 1 hr, partially on the basis of a pre-established list of desirable competences and other characteristics (i.e. fit for role, fit for organization (culture, growth, leadership, long/short term)). The interview time will divided equally - half for the candidate to present themselves, half presenting the Foundation for Public Code. Immediately after the interview both HRM and CEO score the candidate on the items on the list and determine if the candidate should progress to a second interview. HRM will contact the candidate to inform them.
```

Comment (ElenaFdR) [reply activities/recruitment/hiring-process.md]: "The interview time will divided equally - half for the candidate to present themselves, half presenting the Foundation for Public Code."  Is 'to present' an accurate description of how this interview is run and what is expected of the candidate re preparation? 
More conventional first interviews include both potential employer and potential employee having a discussion around pre-prepared questions designed to explore relevant experience, skills and fit - not presenting. A subsequent interview might include a demonstration of skills (eg a presentation to a client, a coding test).

Comment (ericherman) [reply activities/recruitment/hiring-process.md]: I agree that "present" is potentially misleading. Perhaps "introduce" is better?

Comment (ElenaFdR) [reply activities/recruitment/hiring-process.md]: Based on a discussion with @MirjamvT, it sounds like this interview is a shared dialogue to get to know each other, with about half the time devoted to discussing the Foundation (where it's at, how it works, any other questions the candidate may have), and half discussing the candidate (where they're at, what they want next, why they're interested).

**Comment by ericherman** [comment]
@MirjamvT will you please double-check that I resolved the merge conflict correctly?

**Pull request merged**